libs/designer/src/lib/core/state/operation/operationMetadataSlice.ts

```diff
@@ -218,10 +218,6 @@ export const operationMetadataSlice = createSlice({
           state.actionMetadata[id] = actionMetadata;
         }
 
-        if (settings) {
```

Comment:
for some reason we were doing this twice, might have been added extra in a merge